src/main/java/org/primeframework/mvc/security/CipherProvider.java

```diff
@@ -22,13 +22,13 @@
 import java.security.NoSuchAlgorithmException;
 
 /**
- * Provider for getting a Cipher instance that can be used for encryption and any other security as necessary.
+ * Provider for getting a Cipher instance that can be used for AES encryption and any other security as necessary.
```

Comment:
Same comment as below, we shouldn't describe the cipher type, this is up to the implementation.